src/debian/key-script

```diff
@@ -0,0 +1,70 @@
+#! /bin/sh
```

Comment:
I guess you are aware this script needs rewrite as right now it's a copy-paste hybrid from two projects.

I think you have to start from the [Arch version](https://github.com/agherzan/yubikey-full-disk-encryption/blob/76459a9ba9277518e275eca84de1e3f0e1282b46/src/archlinux/hooks/ykfde) then cut-out everything that isn't related to generating secrets (nfc, luks options,trails,timeout, etc.). End result should resemble what debian version does but it needs to be consistent with arch version code syntax.

Reply:
Well, this took a little longer than I thought, but I just finished modifying the Arch Linux version to work with Debian/Ubuntu. I left in the commented out lines from the original for now so it's easier to see what's different between the Arch and Debian versions. I confirmed that it works for both when the Yubikey is present and when there is no Yubikey present.

Reply:
I see you still left some things like nfc (there is no support for it in debian), trials/timeouts (do the work?) and unused variables:
```
$ shellcheck -x key-script

In key-script line 8:
YKFDE_DISK_UUID=""
^-------------^ SC2034 (warning): YKFDE_DISK_UUID appears unused. Verify use (or export if used externally).


In key-script line 9:
YKFDE_LUKS_NAME=""
^-------------^ SC2034 (warning): YKFDE_LUKS_NAME appears unused. Verify use (or export if used externally).


In key-script line 10:
YKFDE_LUKS_DEV=""
^------------^ SC2034 (warning): YKFDE_LUKS_DEV appears unused. Verify use (or export if used externally).


In key-script line 11:
YKFDE_LUKS_OPTIONS=""
^----------------^ SC2034 (warning): YKFDE_LUKS_OPTIONS appears unused. Verify use (or export if used externally).


In key-script line 17:
YKFDE_SLEEP_AFTER_SUCCESSFUL_CRYPTSETUP=""
^-- SC2034 (warning): YKFDE_SLEEP_AFTER_SUCCESSFUL_CRYPTSETUP appears unused. Verify use (or export if used externally).


In key-script line 32:
  local cryptopt cryptoptions
        ^------^ SC2034 (warning): cryptopt appears unused. Verify use (or export if used externally).
                 ^----------^ SC2034 (warning): cryptoptions appears unused. Verify use (or export if used externally).


In key-script line 37:
  . "$YKFDE_CONFIG_FILE" || {
    ^------------------^ SC1091 (info): Not following: ../ykfde.conf: openBinaryFile: does not exist (No such file or directory)

For more information:
  https://www.shellcheck.net/wiki/SC2034 -- YKFDE_DISK_UUID appears unused. V...
  https://www.shellcheck.net/wiki/SC1091 -- Not following: ../ykfde.conf: ope...

```

Reply:
I just removed the commented sections of code, along with the NFC code and unused variables.

As far as the trials and timeout goes, I have confirmed that they work. After the given number of trials, the prompt no longer accepts challenges or passphrases. After the given timeout, the prompt switches from asking for the challenge to asking for a passphrase.

The only issue I've come across is that I was unable to get the messages working. If there is no YubiKey present, the message saying ` > Waiting x for YubiKey` does not appear. Also, I believe the message saying `Remember to touch the device if necessary.` does not appear.

Other than the lack of messages, everything else appears to be working correctly.